indigo_app/views/workflows.py

```diff
@@ -62,10 +46,10 @@ class WorkflowDetailView(WorkflowViewBase, DetailView):
     def get_context_data(self, *args, **kwargs):
         context = super(WorkflowDetailView, self).get_context_data(**kwargs)
 
-        context['task_groups'] = Task.task_columns(['open', 'pending_review'], self.object.tasks.all())
-
-        # TODO: load from API? filter? exclude closed tasks?
-        context['possible_tasks'] = self.country.tasks.exclude(pk__in=[t.id for t in self.object.tasks.all()]).all()
+        tasks = self.object.tasks.all()
+        context['has_tasks'] = bool(tasks)
+        context['task_groups'] = Task.task_columns(['open', 'pending_review'], tasks)
+        context['possible_tasks'] = self.country.tasks.unclosed().exclude(pk__in=[t.id for t in self.object.tasks.all()]).all()
```

Comment:
should this not be `place` over `country`?

Reply:
@goose-life I was trying to be a bit cheeky and allow national-level workflows that cover locality tasks. But I realise this also allows you to have a locality workflow that has tasks from other localities, which is a bit weird. I'll push a change.

Reply:
Pushed elsewhere: https://github.com/laws-africa/indigo/pull/353/commits/30edf384422e49895cd4ade56742d324556dc7e0